packages/ember-cli-fastboot/package.json

```diff
@@ -70,6 +70,7 @@
     "ember-cli-inject-live-reload": "^2.0.2",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.1",
+    "ember-compatibility-helpers": "^1.2.6",
```

Comment:
@backspace this should be in `dependencies` as it's used in run-time by comsumer apps rather than dev time

Reply:
Okay, fixed, thanks!